### ISSUE TITLE
White page after uploading files

### ISSUE THREAD

**Issue by planetetv** [louislam/uptime-kuma#1075]
### ⚠️ Please verify that this bug has NOT been raised before.

- [X] I checked and didn't find similar issue

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/uptime-kuma/security/policy)

### 📝 Describe your problem

Hello there,

Thanks for that awesome tool...
I'm very sorry if my question is a silly one but, I downloaded the zip files and put it on my OVH shared server (a classic server in fact).
When I type the URL of the subdomain I created for Uptime Kuma, I land on a white page.

When I add an .htaccess file with those infos : 

```
<VirtualHost *:443>
  ServerName sub.domain.com
  SSLEngine On
  SSLCertificateFile /path/to/ssl/cert/crt
  SSLCertificateKeyFile /path/to/ssl/key/key
  # Protocol 'h2' is only supported on Apache 2.4.17 or newer.
  Protocols h2 http/1.1

  ProxyPass / http://localhost:3001/
  RewriteEngine on
  RewriteCond %{HTTP:Upgrade} websocket [NC]
  RewriteCond %{HTTP:Connection} upgrade [NC]
  RewriteRule ^/?(.*) "ws://localhost:3001/$1" [P,L]
</VirtualHost>


```

I land on a error 500 page (internal server error).

Can you help me? 
Maybe i did something wrong?

Thanks a lot, and merry xmas to you all! 👍 
=D

### 🐻 Uptime-Kuma Version

1.11.1

### 💻 Operating System and Arch

Apache Server

### 🌐 Browser

Chrome

### 🐋 Docker Version

None

### 🟩 NodeJS Version

None

**Comment by chakflying** [comment]
No part of the normal install procedure should involve manually downloading zip files. Did you follow the install instructions in the front page?

**Comment by planetetv** [comment]
I tried first with SSH, I succeed to installed npm and stuff thanks to the
terminal of my website shared server (OVH) but when I launched the website,
I have the uptime kuma page and logo but a red warning because of a
websocket error.

So, I put an .htaccess with the Apache informations the author put on the «
advanced installation » page but after that I got an error 500.

So I decided to tried by simply upload all the .git files on my server but
of course it does not works that way as it’s not intented to…

That’s why I’m here. Asking for kind help.

So, yes, I followed the install instruction. I’m a college professor, I’m
not used to waste time of people who try to help me.

And, no, I don’t have a Docker service. Just have a normal server from my
university.

Le sam. 18 déc. 2021 à 20:45, Nelson Chan ***@***.***> a
écrit :
